docs/installing/fdo.md

```diff
@@ -59,7 +59,7 @@ Before you purchase FDO-enabled edge devices, you can test FDO support in {{site
    ```
    {: codeblock}
 
-3. Follow the steps in the [FIDO Device Onboard (FDO)](https://github.com/open-horizon/FDO-support/blob/main/README.md){:target="_blank"}{: .externalLink} to observe FDO automatically install the {{site.data.keyword.ieam}} agent on a device and registers it with your {{site.data.keyword.ieam}} management hub.
+3. Follow the steps in the [FIDO Device Onboard (FDO) ](https://github.com/open-horizon/FDO-support/blob/main/README.md){:target="_blank"}{: .externalLink} to observe FDO automatically install the {{site.data.keyword.ieam}} agent on a device and registers it with your {{site.data.keyword.ieam}} management hub.
```

Comment:
Should we link to this document instead?
https://open-horizon.github.io/docs/fdo/README-FDO1.1/

Reply:
The "FIDO Device Onboard (FDO) Protocol Reference Implementation (PRI) Quick Start" is relatively low level.
The current link goes to the FDO README.  This commit just fixes how the External Link glyph and text looks like.

There is longer term discussion on what the FDO landing page and documents should be.  See #439 
At that point, this particular page will be modified.  Suggest we defer the FDO decision to #439